darwin/main.m

```diff
@@ -10,6 +10,41 @@
 static BOOL (^isRunning)(void);
 static BOOL stepsIsRunning;
 
+@interface sizeWrapper : NSView {
```

Comment:
Why is any of this necessary? You also are not properly initializing or uninitializing instances of these objects.